packages/react-native-bpk-component-badge/stories.js

```diff
@@ -96,21 +96,38 @@ const generateBadgeStory = (
       <StorySubheading>{capitalize(i)}</StorySubheading>
       <View style={[badgeWrapperStyle, style[i]]}>
         {contents.map(content => (
-          <BpkBadge
-            key={content}
-            message={content}
-            accessoryView={
-              config.icons ? (
-                <BpkBadgeIcons
-                  icons={iconSets[config.icons]}
-                  separator={config.icons === 'multiple' ? '+' : null}
-                />
-              ) : null
-            }
-            docked={config.docked}
-            type={i}
-            style={style.badge}
-          />
+          <Fragment key={content}>
+            <BpkBadge
+              message={content}
+              accessoryView={
+                config.icons ? (
+                  <BpkBadgeIcons
+                    icons={iconSets[config.icons]}
+                    separator={config.icons === 'multiple' ? '+' : null}
+                  />
+                ) : null
+              }
+              docked={config.docked}
+              type={i}
+              style={style.badge}
+            />
+            {config.icons === 'multiple' && (
+              <BpkBadge
+                // accessibilityLabel={content}
```

Comment:
Commented out?

Reply:
Good catch! I was testing that the prop types warning came up. Pushing an update now.